evap/results/tools.py

```diff
@@ -51,6 +61,37 @@ def mix(a, b, alpha):
     return alpha * a + (1 - alpha) * b
 
 
+def mix_distributions(a, b, alpha):
+    sum_a = sum(a.values())
+    sum_b = sum(b.values())
+    distribution = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
+    if sum_a == 0 and sum_b == 0:
+        return distribution
+    if sum_a == 0:
+        return b
+    if sum_b == 0:
+        return a
+
+    for answer, count in a.items():
+        distribution[answer] += alpha * count
+    for answer, count in b.items():
+        distribution[answer] += (1 - alpha) * count
+    return distribution
+
+
+def expand_to_100_percent(a, b):
+    if a == 0 and b == 0:
+        return 0, 0
+    if a == 0:
+        return 0, 1
+    if b == 0:
+        return 1, 0
```

Comment:
The 2nd and 3rd if statements aren't needed. Is there a reason you'd want to keep it?

Reply:
obsolete :)